src/components/ListItem.tsx

```diff
@@ -8,7 +8,9 @@ import { COLORS } from '../constants/theme/colors'
 
 const Container = styled(Pressable)<{ pressed: boolean }>`
   min-height: ${wp('22%')}px;
-  margin: 0px 16px 8px 16px;
+  width: ${wp('90%')}px;
+  margin-bottom: 8px;
+  align-self: center;
   padding: 12px 8px 12px 16px;
```

Comment:
does this change comply to our designs?

Reply:
I have to merge main and use the changes from Andy. Will do that once #240 is approved/merged